src/adapters/github.js

```diff
@@ -160,19 +160,18 @@ class GitHub extends PjaxAdapter {
     }
 
     const isPR = type === 'pull';
+    const showIn = this.store.get(STORE.SHOWIN);
 
     // Skip rendering the octotree in the unselected pages
-    if (!this.store.get(STORE.SHOWIN_ALL)) {
+    if (showIn && showIn != 0) {
```

Comment:
Not we always use strict equality `===` except for null/undefined check.

Reply:
The reason why i used == was because i used 0 -> 3 for the constant SHOWIN, because the raido input's value is string only...
But since it should be enum instead of 0 -> 3 then i will replace all these with strict equality